CHANGELOG.md

```diff
@@ -12,3 +22,7 @@
   * adds FlattenJSON as default adapter [@joaomdmoura]
```

Comment:
Yeah, that's why I wanted a label for 'add to changelog' for all the times we haven't done it